### PR TITLE
Bug 1165257 - [Ringtones] Migrate to tagged template strings

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1165257

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0c2138cc88e9048f4537e317a2e0fc6630c161e9

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=11895cef2555de27a2e9a1803c7a92e06f9fefc2

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=11895cef2555de27a2e9a1803c7a92e06f9fefc2

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=152078e867eff73acf6632f00a7cb01425f8a6d6

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3720a9707aa2e6a36eb9b791bfbc5d06dd0718ac

File: apps/ringtones/js/manage_template.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+'use strict';
+/* global Tagged */
+(function(exports) {
+  exports.templates = {
```
Comment (jimporter) [inline apps/ringtones/js/manage_template.js]: Please just use `var Templates = { ... };` here (at the global scope). That's more in keeping with the style for the rest of the ringtones app.

File: apps/ringtones/js/manage_template.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+'use strict';
```
Comment (jimporter) [inline apps/ringtones/js/manage_template.js]: Nit: Please rename this file to manage_templates.js (note the plural).

File: apps/ringtones/js/manage_template.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            <span data-l10n-id="${l10nID}"></span>
+          </gaia-subheader>
+          <ul data-type="list"></ul>
+        </section>`;
+    },
+
+    soundItem: function({l10nID, name}) {
+      return Tagged.escapeHTML `
+        <li>
+        <aside class="pack-end">
```
Comment (jimporter) [inline apps/ringtones/js/manage_template.js]: Nit: the indentation doesn't line up here.

File: apps/ringtones/js/manage_template.js (hunk 1 above)
Comment (KevinGrandon) [inline apps/ringtones/js/manage_template.js]: Right now the same page will access the templates blindly. I'll update this to ```var Templates``` for now, and we can revisit it if needed.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5f74bb089454ac01a1f0b056f28ed0abf60876a4

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2e90bf8c1c76c23f54c99f7f6fa4585ceb74bb72

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f24b6a06f6e396653246dc049b6521e33288eb32

**Pull request merged**